ui/src/runtime/composables/useActionExecuter.js

```diff
@@ -0,0 +1,83 @@
+import { useRuntimeConfig } from "#app";
+import { useComposableResolver, useDataFetcher, usePathBuilder } from "#imports";
+
+export default function() {
+  const actions = {
+    "Composite": Composite({ actionExecuter: { execute } }),
+    "Local": Client(),
+    "Remote": Remote()
+  };
+
+  function Client() {
```

Comment:
action implementations should be sorted by their name, above list order and function order should be the same